content/en/cloudcraft/components-aws/network-acl.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Network ACL
+---
+
+## Overview
+
+Use the Network ACL component to visualize network access control lists from your Amazon Web Services architecture.
+
+{{< img src="cloudcraft/components-aws/network-acl/component-network-acl-diagram.png" alt="Screenshot of an isometric Cloudcraft diagram showing the 'Network ACL' AWS component." responsive="true" style="width:60%;">}}
+
+## Toolbar
+
+Use the toolbar to configure and customize the component. The following options are available:
+
+- **Visibility**. Toggle the visibility of the component on the diagram.
+- **Color**. Select a fill color for the body of the component and an accent color for its symbol. You can use the same colors on 2D and 3D views or different colors for each.
+- **Name**. Give the NACL a name.
+- **Shape**. Select a shape for the component.
+- **Padding**. Increase or decrease the amount of space inside the component.
+- **Rules**. Add, edit, or remove inbound and outbound rules for the NACL.
```

Comment:
```suggestion
- **Rules**: Add, edit, or remove inbound and outbound rules for the NACL.
```

Reply:
The rules here are read-only. So actually I would change this to "View inbound and outbound rules for the NACL"